js/views/modals/Settings/Moderation.js

```diff
@@ -1,9 +1,13 @@
-import _ from 'underscore';
+import $ from 'jquery';
+import 'select2';
```

Comment:
Please use our local mac-gyvered select2 from the 'js/lib' folder.

I did remove the 'select2' package from package.json, but it looks like you need to `npm prune` to get it out of there once you've previously installed it. Then, it will flag that import as an issue, which is what we want.